gen-ai-rag-vertex-ai-vector-search/README.md

```diff
@@ -9,8 +9,15 @@ Center](https://cloud.google.com/architecture/).
 To try this sample:
 
   1. [Enable the required APIs](https://console.cloud.google.com/flows/enableapi?apiid=run.googleapis.com,pubsub.googleapis.com,aiplatform.googleapis.com,iam.googleapis.com,storage.googleapis.com,cloudfunctions.googleapis.com,eventarc.googleapis.com,cloudbuild.googleapis.com).
-  1. Follow the [setup instructions](/README.md#setup).
-  1. Make changes to the sample, as indicated by `Note: ` comments.
+  1. Follow the [setup instructions](/README.md#setup) to deploy the sample.
+
+This will deploy the base architecture, but the applications will not be functional.
```

Comment:
maybe instead .. 

This will deploy the supporting infrastructure for the pattern. 

followed with a 

"If you want to install a functional application, you can use X"